### PR TITLE
Add lego-bricks extension

### PR DESCRIPTION
**Pull request by mSarheed**
## Description

Look up LEGO sets, colors, bricks etc.

## Screencast
lego-bricks-1
lego-bricks-2
lego-bricks-3
lego-bricks-4
lego-bricks-5

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/lego-bricks/src/latest-sets.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  useEffect(() => {
+    revalidate();
+  }, [searchText]);
+
+  return (
+    <List
+      isLoading={isLoading}
+      onSearchTextChange={setSearchText}
+      searchBarPlaceholder={isLoading ? "Still loading..." : "Search for LEGO Sets..."}
```
Comment (pernielsentikaer) [inline extensions/lego-bricks/src/latest-sets.tsx:43]: Avoid using `searchBarPlaceholder` for loading text, `isLoading` is the place 🙂

File: extensions/lego-bricks/package.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    },
+    {
+      "name": "parts",
+      "title": "Parts",
+      "description": "Show a list of all LEGO parts",
+      "mode": "view"
+    },
+    {
+      "name": "latest-sets",
+      "title": "Latest LEGO sets releases",
```
Comment (pernielsentikaer) [inline extensions/lego-bricks/package.json:41]:
```suggestion
      "title": "Show Latest LEGO Sets Releases",
```

File: extensions/lego-bricks/package.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  "categories": [
+    "Fun",
+    "Design Tools",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "colors",
+      "title": "Colors",
```
Comment (pernielsentikaer) [inline extensions/lego-bricks/package.json:17]:
```suggestion
      "title": "Show Colors",
```

File: extensions/lego-bricks/package.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      "description": "Show a list of all LEGO colors",
+      "mode": "view"
+    },
+    {
+      "name": "minifigs",
+      "title": "Mini Figures",
```
Comment (pernielsentikaer) [inline extensions/lego-bricks/package.json:23]:
```suggestion
      "title": "Show Mini Figures",
```

File: extensions/lego-bricks/package.json [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+      "description": "Show a list of all LEGO mini figures",
+      "mode": "view"
+    },
+    {
+      "name": "sets",
+      "title": "Sets",
```
Comment (pernielsentikaer) [inline extensions/lego-bricks/package.json:29]:
```suggestion
      "title": "Show Sets",
```

File: extensions/lego-bricks/package.json [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+      "description": "Show a list of all LEGO sets",
+      "mode": "view"
+    },
+    {
+      "name": "parts",
+      "title": "Parts",
```
Comment (pernielsentikaer) [inline extensions/lego-bricks/package.json:35]:
```suggestion
      "title": "Show Parts",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by mSarheed** [comment]
@pernielsentikaer  Thanks for the comments. Suggested changes have been made, please review :)

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**